libs/nx-react-workshop/project.json

```diff
@@ -49,6 +49,11 @@
             "input": "./libs/nx-react-workshop",
             "glob": "migrations.json",
             "output": "."
+          },
+          {
+            "input": "./examples",
+            "glob": "**/*",
+            "output": "./examples"
           }
```

Comment:
This ensures the example files available for use in generators + migrations allow us to remove the duplicate code that is currently inlined into the migrations.